virt/virt-web-console-overview.adoc

```diff
@@ -544,8 +544,8 @@ The number of CPUs is calculated by using the following formula: `sockets * thre
 |SSH using virtctl
 |Click the copy icon to copy the `virtctl ssh` command to the clipboard.
 
-|SSH over NodePort
-|Selecting *Create a Service to expose your VirtualMachine for SSH access* generates an `ssh -p <port>` command. Click the copy icon to copy the command to the clipboard.
+|SSH service type options
+|Select the option for *SSH using LoadBalancer* or *SSH over NodePort*. Both options generate an `ssh -p <port>` command. Click the copy icon to copy the command to the clipboard.
```

Comment:
We want to keep the web overview as concise as possible. One way is by not documenting procedures in the description. 

I recommend that you change this to  `Select *SSH over LoadBalancer* or *SSH over NodePort*.`
The rest will be obvious to the user.